src/sources/util/http/prelude.rs

```diff
@@ -186,6 +190,7 @@ pub trait HttpSource: Clone + Send + Sync + 'static {
 
             let span = Span::current();
             let make_svc = make_service_fn(move |conn: &MaybeTlsIncomingStream<TcpStream>| {
+                let remote_addr = conn.peer_addr();
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I'd suggest either removing this var and calling it directly in the `PeerAddr` creation, or cloning this var on line `200`.